packages/edit-post/src/components/sidebar/post-link/style.scss

```diff
@@ -9,3 +9,10 @@
 .edit-post-post-link__link {
 	word-wrap: break-word;
 }
+
+/* rtl:begin:ignore */
+body.rtl .edit-post-post-link__preview-link-container {
```

Comment:
Ideally, we don't rely on `body.rtl` as these components can be shown anywhere and not only the editor pages. One way of doing this is to rely on the `isRTL` of the editor settings. But I'd like to understand why the default RTL handling was not working properly?

Reply:
> Ideally, we don't rely on `body.rtl` as these components can be shown anywhere and not only the editor pages.
True I will update to use a class conditionally added by isRTL.

> But I'd like to understand why the default RTL handling was not working properly?

The default handling was working properly the problem we have is that it looks like even on RTL languages the permalink is not RTL so we trying to force LTR on this zone.

Reply:
Is there a way we could solve this just by using `/* rtl:begin:ignore */` as I assume these styles could apply similarily to both directions right?

Reply:
> Is there a way we could solve this just by using `/* rtl:begin:ignore */` as I assume these styles could apply similarily to both directions right?

The problem is that direction rtl is inherited so the direction elements are rendered is changed and we need a rule to overwrite the direction in a specific zone.
I also verified that simply adding an ignore on the zone long-content-fade is used does not produces the ideal result. Probably that's the reason long-content-fade receives a direction parameter.

Reply:
Can we force the direction here. Basically what I'm saying is: Can't we just keep these styles you added but remove  `body.rtl`?

Reply:
Hi Riad, I updated this PR going in the direction you are suggesting, we now don't use body.rtl. Thank you for the feedback 👍